pkg/apis/application/v1alpha1/applicationset_types.go

```diff
@@ -87,11 +87,38 @@ type ApplicationMatchExpression struct {
 	Values   []string `json:"values,omitempty" protobuf:"bytes,3,opt,name=values"`
 }
 
+// ApplicationsSyncPolicy representation
+// "create-only" means applications are only created. If the generator's result contains update, applications won't be updated
+// "create-update" means applications are only created/Updated. If the generator's result contains update, applications will be updated, but not deleted
+// "create-delete" means applications are only created/deleted. If the generator's result contains update, applications won't be updated, if it results in deleted applications, the applications will be deleted
+// "sync" means create/update/deleted. If the generator's result contains update, applications will be updated, if it results in deleted applications, the applications will be deleted
```

Comment:
Let's add a statement saying, "If no ApplicationsSyncPolicy is defined, it defaults it to `sync`". WDYT?

Reply:
Better you are right